api/graphql/tests/test_reservation_units.py

```diff
@@ -883,6 +883,182 @@ def test_filtering_by_is_draft_false(self):
+        )
+        response = self.query(
+            """
+            query {
+                reservationUnits(orderBy: "nameSv") {
+                    edges {
+                        node {
+                            nameSv
+                        }
+                    }
+                }
+            }
+            """
+        )
+        content = json.loads(response.content)
+        assert_that(content.get("errors")).is_none()
+        self.assertMatchSnapshot(content)
+
+    def test_order_by_type_fi(self):
+        res_type = ReservationUnitTypeFactory(name="name_fi", name_fi="name_fi")
+        ReservationUnitFactory(
+            reservation_unit_type=res_type,
+        )
+        response = self.query(
+            """
+            query {
+                reservationUnits(orderBy: "typeFi") {
+                    edges {
+                        node {
+                            reservationUnitType {nameFi}
+                        }
+                    }
+                }
+            }
+            """
+        )
+        content = json.loads(response.content)
+        assert_that(content.get("errors")).is_none()
+        self.assertMatchSnapshot(content)
+
+    def test_order_by_type_en(self):
+        res_type = ReservationUnitTypeFactory(name="name_en", name_fi="name_en")
+        ReservationUnitFactory(
+            reservation_unit_type=res_type,
+        )
+        response = self.query(
+            """
+            query {
+                reservationUnits(orderBy: "typeEn") {
+                    edges {
+                        node {
+                            reservationUnitType {nameEn}
+                        }
+                    }
+                }
+            }
+            """
+        )
+        content = json.loads(response.content)
+        assert_that(content.get("errors")).is_none()
+        self.assertMatchSnapshot(content)
+
+    def test_order_by_type_sv(self):
+        res_type = ReservationUnitTypeFactory(name="name_sv", name_fi="name_sv")
+        ReservationUnitFactory(
+            reservation_unit_type=res_type,
+        )
+        response = self.query(
+            """
+            query {
+                reservationUnits(orderBy: "typeSv") {
+                    edges {
+                        node {
+                            reservationUnitType {nameSv}
+                        }
+                    }
+                }
+            }
+            """
+        )
+        content = json.loads(response.content)
+        assert_that(content.get("errors")).is_none()
+        self.assertMatchSnapshot(content)
+
+    def test_order_by_unit(self):
+        unit = UnitFactory(id=9999)
+        ReservationUnitFactory(
+            unit=unit,
+        )
+        response = self.query(
+            """
+            query {
+                reservationUnits(orderBy: "unit") {
+                    edges {
+                        node {
+                            unit {pk}
```

Comment:
Let's use `nameFi` or some other constant field instead of `pk` to make the snapshots deterministic. 🙂 Same in `test_order_by_unit_reverse_order` below.

Reply:
fixed